content/news/XXXX-XX-XX-github-issues-migration.md

```diff
@@ -0,0 +1,41 @@
+title: Migrated to Github issues
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Yes everybody, we're officially moving away from Launchpad to Github.
+If you've never tried to report a Bug in Mixxx or wanted to contribute
+otherwise, its very likely you have no idea what this Post is about,
+but if you do its even better. No matter if you do or don't, keep reading.
+
+The mixxx team has traditionally been using canonicals (the makers of Ubuntu)
+launchpad platform for hosting their code, managing bug reports and releases.  
+Over the years, launchpad has become increasingly dated, lack of modern features,
+old UI, and bad user experience is the primary reason why many people (myself
+included), especially new potential contributors, stay away from launchpad.
+We suspect Launchpad to be one of the major deterents for potential mixxx contributors.
+We've started transitioning away from launchpad quite a while ago.
+<!-- The rest of this paragraph needs further info from older mixxx team members.
+As I was not around when this happened, most of what I'm writing here is pure speculation
+from my part. Please correct me if I'm wrong or if I should leave this out completely.-->
+
+Back in XXXX we made the first major step away from Launchpad by migrating the
+mixxx source code to github. Still, bug reports continued to be handled on launchpad.
+TODO
+
+TODO launchpad just used for the ppa, is that correct? no contributor interaction on launchpad
+anymore.
+
+TODO write about how we migrated all issues to mixxx
+
+# What does this mean for existing contributors?
+
+From today on, no more bug reports should be filed on launchpad. Currently on-going
+discussion should be continued on Github instead. Note that any comments made
+after 2022-07-07 22:00 UTC have not been migrated to Github.
+<!-- That's when I scraped LP last -->
```

Comment:
We can make them read only before the final scrape. On the other hand it should be also not much of a problem to add the missing bits.